### PR TITLE
Visualize the polygons

### PR DESCRIPTION
**Pull request by im-prakher**
I have visualized the polygons as they should have been. Also modified the tests for that.
But in this PR I don't know why GitHub is failing to recognize the previous merged commits. As mentioned in this issue, [https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch](https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch)
I have already tried different methods to solve this but that didn't work for this.
Kindly, review this @Chris-Schnaufer @dlebauer @KristinaRiemer

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
I'm not sure why the previously merged commits are included here. The hashes are different for the same commits for some reason, see an example below: 

[PR](https://github.com/PecanProject/BETYdb-YABA/pull/68/commits/323291d10fa92ab6168fa292d5c410967dd54aa7):
Screen Shot 2020-08-26 at 10 47 22 AM

[Merged](https://github.com/PecanProject/BETYdb-YABA/commit/688a537daf2b51bc05c8ccc2174fff58413d1b43):
Screen Shot 2020-08-26 at 10 46 44 AM

**Comment by KristinaRiemer** [comment]
@im-prakher is this branch called `tests` a different branch than the previous ones called `tests` that had the tests in it (https://github.com/PecanProject/BETYdb-YABA/pull/60)?

**Comment by im-prakher** [comment]
No! It is the same branch

**Comment by KristinaRiemer** [comment]
After selecting all the relevant `input_files` files on the upload pages, I got this error on the cultivars map page: 
Screen Shot 2020-08-28 at 7 46 11 AM

File: interface/src/components/homepage.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ class homepage extends Component{
                 <div className="steps">
                     <div><span>Steps</span></div>
                     <ol>
-                        <li>Fill out Metadata in this google spreadsheet template (How to do this).</li>
-                        <li>Prepare a shapefile or table with plot polygons (How to do this).</li>
+                        <li>Fill out Metadata in the google spreadsheet template (<a href="https://docs.google.com/spreadsheets/d/1c_5j7q3TO6gQ24KSopE-_LXA5HhfsUEqMupckGZAbo8/edit#gid=0" rel="noopener noreferrer" target="_blank" style= { { color: "blue"} }>How to do this</a>).</li>
```
Comment (Chris-Schnaufer) [inline interface/src/components/homepage.js:19]: Maybe keep the color to what the user prefers - could impact Blue color blind people. Another possibility is to use the aria role `link` with css: https://www.w3.org/TR/wai-aria-1.1/#link

File: visualization/pg_joins.js [hunk 2]
```diff
@@ -52,40 +52,67 @@ const fetch_specieId= async (client, specie)=>{
...
 }
 
+const getGeometry= async (client, geometry)=>{
+  try{
+    geometry.coordinates[0].map((data)=>{
+      if(data.length<3)
+        data.push(115);
+      return data
+    })
+    const res= (await client.query('SELECT ST_Geomfromtext(ST_AsText(ST_GeomFromGeoJSON($1)),4326) As wkt', [geometry]));
```
Comment (Chris-Schnaufer) [inline visualization/pg_joins.js:62]: Nice use of parameters for SQL statements!

File: interface/src/components/homepage.js (hunk 1 above)
Comment (dlebauer) [reply interface/src/components/homepage.js:19]: isn't this just for a hyperlink? From quick research, it looks like blue hyperlinks can be distinguished from black text. e.g. [this site](https://gomakethings.com/better-link-accessibility-for-the-color-blind) says "The blue links on this site are easily distinguishable from plain text to all three forms of color blindness."

For the plot colors, you can find color-blind friendly palettes from here: https://colorbrewer2.org/#type=diverging&scheme=PuOr&n=5

File: visualization/pg_joins.js (hunk 2 above)
Comment (im-prakher) [reply visualization/pg_joins.js:62]: Thank you!

**Comment by im-prakher** [comment]
The changes in this PR are reflected in #69 so I am closing this one.

**Comment by KristinaRiemer** [comment]
@im-prakher I think this needs to be merged instead of closed?

**Pull request closed without merging**